src/Aspire.Hosting/Seq/SeqBuilderExtensions.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Seq server resources to the application model.
+/// </summary>
+public static class SeqBuilderExtensions
+{
+    // The path within the container in which Seq stores its data
+    const string SeqContainerDataDirectory = "/data";
+
+    /// <summary>
+    /// Adds a Seq server resource to the application model. A container is used for local development.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name to give the resource.</param>
+    /// <param name="port">The host port for the Seq server.</param>
+    /// <param name="seqDataDirectory">Host directory to bind to Seq's data directory. This must already exist.</param>
+    public static IResourceBuilder<SeqResource> AddSeq(
+        this IDistributedApplicationBuilder builder,
+        string name = "Seq",
+        int port = 5341,
```

Comment:
Why fix the host port? It'll end up conflicting pretty easily.

Reply:
fixed